File: python/fileupload.py
import time
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.common.desired_capabilities import DesiredCapabilities

desired_cap = {
    'browserName': 'firefox',
    'javascriptEnabled': True,
    'platform': 'WIN8',
    'version': '', 
}

driver = webdriver.Remote(
	command_executor='http://BS_USERNAME:BS_AUTHKEY@hub.browserstack.com/wd/hub',
	desired_capabilities=desired_cap)

driver.get("http://www.fileconvoy.com/")
driver.find_element_by_id("upfile_0").send_keys("C:\\Users\\hello\\url.txt")
driver.find_element_by_id("readTermsOfUse").click()
driver.find_element_by_name("form_upload").submit()
time.sleep(5)
print(driver.find_element_by_tag_name("center").text)
driver.quit()

File: csharp/FileUpload.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Remote;

namespace ConsoleApplication1
{
  class Program
  {
    static void Main(string[] args)
    {
      DesiredCapabilities capabilities = DesiredCapabilities.Firefox();
      capabilities.SetCapability("os", "windows");
      capabilities.SetCapability("os_version", "8");
      capabilities.SetCapability("browserstack.user", "BS_USERNAME");
      capabilities.SetCapability("browserstack.key", "BS_AUTHKEY");

      IWebDriver driver = new RemoteWebDriver(new Uri("http://hub.browserstack.com/wd/hub/"), capabilities);

      driver.Navigate().GoToUrl("http://www.fileconvoy.com/");

      driver.FindElement(By.Id("upfile_0")).SendKeys("C:\\Users\\hello\\url.txt");
      driver.FindElement(By.Id("readTermsOfUse")).Click();
      driver.FindElement(By.Name("form_upload")).Submit();

      Console.Out.WriteLine(driver.FindElement(By.TagName("center")).Text);

      driver.Quit();
    }
  }
}

